src/compile/scale/domain.ts

```diff
@@ -431,10 +423,10 @@ export function getFieldFromDomain(domain: VgDomain): string {
     }
     log.warn('Detected faceted independent scales that union domain of identical fields from different source detected.  We will assume that this is the same field from a different fork of the same data source.  However, if this is not case, the result view size maybe incorrect.');
     return field;
-  } else if (isFieldRefUnionDomain(domain) && isString) {
+  } else if (isFieldRefUnionDomain(domain) && util.isString(domain)) {
```

Comment:
@kanitw Note that this was definitely wrong before.

Reply:
lol